recognition/scoring.py

```diff
@@ -44,14 +45,42 @@ def run(self):
 
 
 class ScliteJob(Job):
-    def __init__(self, ref, hyp, cer=False, sort_files=False, additional_args=None):
+    """
+    Run the Sclite scorer from the SCTK toolkit
+
+    Outputs:
+        - out_report_dir: contains the report files with detailed scoring information
+        - out_*: the job also outputs many variables, please look in the init code for a list
+    """
+
+    def __init__(
+        self,
+        ref: tk.Path,
+        hyp: tk.Path,
```

Comment:
Is this compatible with the OptimizeAMandLMScaleJob ? I am not sure that this one inputs tk.Path objects
https://github.com/rwth-i6/i6_core/blob/8dab653e8c6d6e872b3b0be69ad3307186aeb915/recognition/optimize_parameters.py#L99-L102

Reply:
What about just using tk.Path in the optimizer?

Reply:
the required changes should be part of this PR or this PR can only land after another PR addressing the changes in the optimizer job.

Reply:
but they are part of this PR!